boards/xtensa/intel_s1000_crb/intel_s1000_crb_defconfig

```diff
@@ -54,3 +54,6 @@ CONFIG_AUDIO_CODEC=y
 CONFIG_AUDIO_TLV320DAC=y
 CONFIG_AUDIO_DMIC=y
 CONFIG_AUDIO_INTEL_DMIC=y
+
+CONFIG_NEURAL_NET=y
```

Comment:
IMHO, that's too generic. If this is an option to enable neural net **accelerator** **drivers**, then the most important thing about it is that it's *hardware driver*, and that's completely not mirrored in the option name.

Reply:
How about `CONFIG_NEURAL_NET_ACCEL`

Reply:
Definitely better, +1 from me.

Reply:
Thanks. Updated to `CONFIG_NEURAL_NET_ACCEL`.

Reply:
Thanks too! So, I'm not qualified to review the rest of the patch to approve it, but informally, +1, it's great to see such cool features added to Zephyr.